include/modules/meta/inotify_module.hpp

```diff
@@ -8,16 +8,7 @@ POLYBAR_NS
 namespace modules {
   template <class Impl>
   class inotify_module : public module<Impl> {
-   public:
-    using module<Impl>::module;
-
-    void start() {
-      this->m_mainthread = thread(&inotify_module::runner, this);
-    }
-
-   protected:
-    void runner() {
-      this->m_log.trace("%s: Thread id = %i", this->name(), concurrency_util::thread_id(this_thread::get_id()));
+   public: using module<Impl>::module; void start() { this->m_mainthread = thread(&inotify_module::runner, this); } protected: void runner() { this->m_log.trace("%s: Thread id = %i", this->name(), concurrency_util::thread_id(this_thread::get_id()));
```

Comment:
Leave the indentation as it was, it is difficult to review and, possibly, introduces useless changes.

Reply:
Strange that it was all put on the same line. Did `clang-format` do that?